src/lightning-stubs/platformShowToastEvent/platformShowToastEvent.js

```diff
@@ -4,14 +4,15 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-const ShowToastEventName = 'lightning__showtoast';
+export const ShowToastEventName = 'lightning__showtoast';
 
 export class ShowToastEvent extends CustomEvent {
-    constructor() {
+    constructor(toast) {
```

Comment:
```suggestion
    constructor(detail) {
```

Reply:
Note to reviewers: If this line changes to match the suggestion, so will line 15